### ISSUE TITLE
Two and more colours in one module

### ISSUE THREAD

**Issue by asmikhailov** [enkore/i3pystatus#286]
Is it possible combine two or more colours in output from one module?

**Comment by facetoe** [comment]
I think this is possible using the Pango markup hint - http://docs.enkore.de/i3pystatus/configuration.html#hints

**Comment by asmikhailov** [comment]
Thank you. You are help me a lot.

**Comment by facetoe** [comment]
No worries!

**Comment by mxlian** [comment]
Hi @enkore, what should be added to the docs? I've look at them and they seem quite self explaining...

**Comment by lasers** [comment]
>I've look at them and they seem quite self explaining...

I agree. I tried this and got three colors working okay on `mpd`. Can we close this issue?
